src/Contexts/ModalContext/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { createContext, useContext, useState } from 'react'
+
+const ModalContext = createContext();
+
+const ModalProvider = ({children}) => {
+  const [isModalVisible,setIsModalVisible] = useState(false);
+
+  const hideModal = () => {
```

Comment:
Suggestion: Can reduce the lines, since only one line is required.

Reply:
fixed